src/components/projects/view/moderator-activity-card.svelte

```diff
@@ -0,0 +1,160 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import Accordion from 'ui/accordion.svelte';
+  import Button from 'ui/button.svelte';
+  import Card from 'ui/card.svelte';
+  import Labeled from 'ui/labeled.svelte';
+  import UnclickableChip from 'ui/unclickable-chip.svelte';
+  import ApplicationTile from '@/components/projects/view/application-tile.svelte';
+  import ApplicationStatuses from '@/constants/backend/application-statuses.js';
+  import s from '@/utils/plural-s.js';
+  import { formatDate, formatTimeRange } from '@/utils/date-time-format.js';
+
+  export let activity;
+  export let competences;
+
+  $: pendingApplications = (
+    activity.applications == null ?
+    [] : activity.applications.filter(x => x.status === ApplicationStatuses.PENDING)
```

Comment:
Convention when using ternary operator on multiple lines appears to be something like this:
```suggestion
    activity.applications == null
      ? []
      : activity.applications.filter(x => x.status === ApplicationStatuses.PENDING)
```

Reply:
Idk, I find it excessive, besides, this way kind of keeps the `?` at the line of the condition, which makes more sense to me

Reply:
I think it improves readability, making it look more similar to `if ... else` (by giving each branch a different line) and aligning both lines by starting each with a symbol (`?` and `:` respectively)

Reply:
We might have to agree to disagree on this